packages/start-plugin-core/tests/vite-start-compiler-plugin.test.ts

```
@@ -0,0 +1,139 @@
+import { describe, expect, test, vi } from 'vitest'
+
+vi.mock('@tanstack/start-server-core', () => ({
+  VIRTUAL_MODULES: {
+    serverFnResolver: 'virtual:tanstack-start-server-fn-resolver',
+  },
+}))
+
+import {
+  SERVER_FN_LOOKUP,
+  startCompilerPlugin,
+} from '../src/vite/start-compiler-plugin/plugin'
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)